mitiq/interface/mitiq_qiskit/qiskit_utils.py

```diff
@@ -109,15 +109,16 @@ def execute_with_noise(
         basis_gates = noise_model.basis_gates + ["save_density_matrix"]
 
     # execution of the experiment
-    job = qiskit.execute(
+    backend = AerSimulator(method="density_matrix", noise_model=noise_model)
+    exec_circuit = qiskit.transpile(
         circ,
-        backend=AerSimulator(method="density_matrix", noise_model=noise_model),
-        basis_gates=basis_gates,
+        backend=backend,
         # we want all gates to be actually applied,
         # so we skip any circuit optimization
+        basis_gates=basis_gates,
         optimization_level=0,
```

Comment:
I think the comment should go directly above `optimization_level` for clarity.

Reply:
I went ahead and did the same thing in a few other places